libs/types/src/lib/types.ts

```diff
@@ -31,8 +31,8 @@ export const EventRestrictionMap = ['Everyone', 'Friends', 'Invite Only'];
 export interface IEvent {
   id: number;
   title: string;
-  startDate: Date;
-  endDate: Date;
+  startDate: Date | string;
+  endDate: Date | string;
```

Comment:
Do you really think this is a good idea? I'd rather not have to deal with composites types. It lead to some messy code in minicap.

Reply:
Agreed, this was the one part I wasn't sure about. Let's continue this discussion above https://github.com/RGPosadas/Mull/pull/129#discussion_r548312856

Reply:
I addressed this by adding a separate `ISeralizedEvent` interface to be used for the serialized event. I think this is the best solution since apollo client does not support automatic reverse seralization at the moment as per their [docs](https://www.apollographql.com/docs/apollo-server/schema/scalars-enums/#custom-scalars).

Reply:
I'm fine with this. I realized that my suggestion to update the IEvent interface to have `startDate` and `endDate` as strings might not be possible. Can you confirm that it's not possible, or did you prefer to have a separate interface for this?

Reply:
Yes exactly, the `IEvent` is used in our API so I had to make a separate interface for our front-end

Reply:
Cool, sorry for being so insistent on that solution despite it not really being feasible haha.